d2bs/kolbot/libs/bots/Duriel.js

```diff
@@ -42,11 +42,26 @@ function Duriel() {
 		return target.dead;
 	};
 
+	// return the script position in the list of scripts to be ran
+	this.scriptIndex = function (script) {
```

Comment:
add a feature in Loader.js where the for loop for loading scripts occurs, you can save the index as a global in the loader object and add some function to get next script or get script by index + 1